src/rpc/tokens.rs

```diff
@@ -185,27 +185,3 @@ pub struct TokenIfResponse {
 
 /// Response for Token Info requests
 pub type TokenInfoResponse = Paginated<TokenIfResponse>;
-
-#[cfg(test)]
-mod test {
```

Comment:
a note on this:

i also saw this on the `msig_history` file—i think it would be cool to set up these files on a folder on the repo to keep these tests from breaking ci in another pr! happy to take on that as well

Reply:
yeah, they should be in an integration `tests/` folder but i was lazy